exporter/prometheusremotewriteexporter/helper_test.go

```diff
@@ -284,10 +284,10 @@ func Test_getPromMetricName(t *testing.T) {
 			validDoubleGauge,
 		},
 		{
-			"total_suffix",
+			"no_counter_suffix",
```

Comment:
Consider linking to this PR or adding a comment, this test will be quite confusing to a future reader given there's no code that it reflects.